### PR TITLE
docs: update websocket example with new example props

### PR DESCRIPTION
**Pull request by derberg**
as a followup of changes proposed by Laurent

### CODE REVIEW

**Comment by derberg** [comment]
@magicmatatjahu yo, here you have a real example that you can test against changes in markdown and html (react) templates

File: examples/websocket-gemini.yml [hunk 1]
```diff
@@ -111,7 +111,9 @@ components:
       payload:
         $ref: '#/components/schemas/market'
       examples:
-        - payload:
+        - name: updateMessage
+          summary: Example of an update message hat contains a change in price information.
```
Comment (magicmatatjahu) [inline examples/websocket-gemini.yml:115]: Example of an update message **hat** 

I know that my english isn't good, but `message hat` Wat?! 😄

Comment (derberg) [reply examples/websocket-gemini.yml]: yeah, it is another word, like a synonym for `envelope` (you need to read more about CloudEvents). But yeah, I will switch to `that` this time 😆

Comment (magicmatatjahu) [reply examples/websocket-gemini.yml]: Thanks for clarification!

**Review by magicmatatjahu — APPROVED** [review]

**Pull request merged**